portage-git.Dockerfile

```diff
@@ -0,0 +1,26 @@
+# This Dockerfile creates a shallow ::gentoo git snapshot that can be
+# mounted as a container volume. It utilizes a multi-stage build and
+# requires docker-17.05.0 or later. It performs a shallow clone of the
+# sync friendly ::gentoo git repository from the official sources and
+# verifies its checksum as well as its gpg signature.
+
+FROM --platform=$BUILDPLATFORM alpine:3.11 as builder
+
+ARG GIT_REPO="https://github.com/gentoo-mirror/gentoo.git"
+ARG SIGNING_KEY="F748E9B3C47E393CC24C8FAF7C2AC09CD98F2EDF"
+
+RUN apk add --no-cache ca-certificates gnupg git
+RUN	git clone --depth 1 "${GIT_REPO}" /gentoo-git
```

Comment:
is this extra space intentional?

Reply:
No, that was a stray tab character. Thanks for pointing it out.